src/mad_spark_alt/core/multi_perspective_prompts.py

```
@@ -0,0 +1,233 @@
+class MultiPerspectivePrompts:
+    """Perspective-specific prompts for QADI analysis."""
+    
+    def __init__(self) -> None:
+        """Initialize with perspective-specific prompt templates."""
+        self.perspective_prompts = {
+            QuestionIntent.ENVIRONMENTAL: {
+                "role": "environmental sustainability expert",
+                "question_focus": "ecological impact and sustainable solutions",
+                "hypothesis_focus": "environmentally responsible approaches",
+                "evaluation_criteria": "environmental benefit, feasibility, and long-term sustainability",
+                "verification_focus": "real-world environmental success stories"
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)